src/delivery/forms.py

```diff
@@ -15,9 +15,24 @@ class DishIngredientsForm(forms.Form):
     )
 
     ingredients = forms.ModelMultipleChoiceField(
-        label=_('Select Ingredients:'),
+        label=_('Select Main Dish Ingredients:'),
```

Comment:
No need to use camel case here. Just: _Select main dish ingredients_ should be fine.